README.md

```diff
@@ -713,8 +715,17 @@ def test_pebble_exec():
         container.pebble_ready_event,
         state_in,
     )
+    assert state_out.containers["foo"].get_exec(('ls', '-ll')).stdin = "..."
 ```
 
+Scenario will attempt to find the right `Exec` object by matching the provided
+command prefix against the command used in the ops `container.exec()` call. For
+example, if the command is `['ls', '-ll']` then Scenario will look for an `Exec`
+with exactly the same as command prefix, `('ls', '-ll')`, and if not found will
+look for an `Exec` with the command prefix `('ls', )`, and if not found will
+look for an `Exec` with the command prefix `()`, and if not found will raise
+a `RuntimeError`.
+
```

Comment:
Consider documenting that it is an error to supply repeated prefixes.

Either that or that the order is not significant, though that should be obvious, right?

Reply:
P.S. this block may read better if the it were a bullet-point list, like:
* will try abcd
* will try abc
* will try ab
* raise and error